Tasks/GooglePlayReleaseV4/main.ts

```diff
@@ -69,6 +69,9 @@ async function run(): Promise<void> {
         const uploadMappingFile: boolean = tl.getBoolInput('shouldUploadMappingFile', false) && (action === 'SingleApk' || action === 'SingleBundle');
         const mappingFilePattern: string = tl.getInput('mappingFilePath');
 
+        const uploadNativeMappingFile: boolean = tl.getBoolInput('shouldUploadNativeMappingFile', false) && (action === 'SingleApk' || action === 'SingleBundle');
+        const nativeMappingFilePattern: string = tl.getInput('nativeMappingFilePath');
```

Comment:
Could you add information about the new task inputs to the documentation?

Reply:
Added in https://github.com/microsoft/google-play-vsts-extension/pull/330/commits/5e97e0a2151789e78228cebac93e79914626b242